buttons/button_base.h

```diff
@@ -30,27 +31,36 @@ class ButtonBase : public Looper,
         current_modifiers |= button_;
       }
       while (DebouncedRead()) {
-	if (millis() - push_millis_ > 300) {
-	  prop.Event(button_, EVENT_HELD);
-	  while (DebouncedRead()) {
-	    if (millis() - push_millis_ > 2000) {
-	      prop.Event(button_, EVENT_HELD_LONG);
-	      while (DebouncedRead()) YIELD();
-	    }
-	    YIELD();
-	  }
-	}
-	YIELD();
-      }
-      while (DebouncedRead()) YIELD();
-      prop.Event(button_, EVENT_RELEASED);
-      if (current_modifiers & button_) {
-        current_modifiers &=~ button_;
-        if (millis() - push_millis_ < 500) {
-          prop.Event(button_, EVENT_CLICK_SHORT);
-        } else {
-          prop.Event(button_, EVENT_CLICK_LONG);
+          if (!short_click){
```

Comment:
short click will ALWAYS be false here

Reply:
No, short_click is set to true when a subsequent short click occurs within 500 ms of the first.  this way if we are a bit long on the press but release and press the button again (like one does when spamming blaster blocks) the event_held doesn't fire and we dont' get false held events.  I just did a test with line 60 commented out vs not: when short_click is set to true, the false event_held events don't show up on the console.  When that line is commented and short_click is always false, event_held events are shown in the log.

Reply:
well that's embarrassing.  Could have sworn it made a difference in testing but just checked again with some prints and it never flags true.  Removing.